Tasks/AzureKeyVaultV2/task.json

```diff
@@ -14,7 +14,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 189,
+        "Minor": 192,
```

Comment:
@50Wliu  Could you please bump the minor version of the task? 
Also, seems that changes from task.lock.json file is missing. Could you please update it as well?
 You can do this following this guide: https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/taskversionbumping.md

Reply:
Sure - just updated all of them to minor version 199.